backend/bot/index.js

```diff
@@ -14,7 +29,23 @@ bot.on('start', async () => {
   }
 });
 
-const postResultToSlack = (match) => {
+const updateSlack = async (match) => {
```

Comment:
this function is definitely too long and the resopnsilibities are the following:
1. update channel purpose
2. print slack message, augmented if comedy duo plays 

pls break it down